manifests/eap/client/modulepath.pp

```diff
@@ -1,18 +1,26 @@
 # Setting up common module path for lightblue::eap::client resource
 class lightblue::eap::client::modulepath {
 
-    # Pretty sure many of those paths will conflict with something else...
-    # Perhaps better use mkdir -p ?
-    $module_base = [ '/usr/share/jbossas/modules/com',
-        '/usr/share/jbossas/modules/com/redhat',
-        '/usr/share/jbossas/modules/com/redhat/lightblue',
-        '/usr/share/jbossas/modules/com/redhat/lightblue/client']
+    $module_path = '/usr/share/jbossas/modules/com/redhat/lightblue/client'
 
-    file { $module_base :
-        ensure   => 'directory',
-        owner    => 'jboss',
-        group    => 'jboss',
-        mode     => '0755',
+    exec { $module_path:
+        command => "mkdir -p $module_path",
+        user    => 'jboss',
```

Comment:
You're right. Certificates and passwords should not be readable to anyone, like other jboss modules. Pushed a fix.